scripts/export-upm-package.sh

```diff
@@ -8,9 +8,22 @@ upm_name=org.khronos.UnityGLTF
 upm_src_folder_path=$(pwd)/UnityGLTF/Assets/UnityGLTF
 upm_manifest_path=$(pwd)/scripts/package.json
 upm_staging_path=$(pwd)/current-package/$upm_name
+upm_staging_UWP_plugins_path=$upm_staging_path/UnityGLTF/Plugins/uap10.0.10586
 upm_zip_export_path=$(pwd)/current-package/$upm_name.zip
 upm_targz_export_path=$(pwd)/current-package/$upm_name.tar.gz
 
+# msbuild spits out every single dependency dll for UWP
+# These are the only files that are needed by the UPM package for Unity 2018.3+
+# Including all the files in the UWP plugin directory causes name collision errors when 
+# building for UWP in Unity
+upm_UWP_Plugins=(
```

Comment:
@ei2kpi-ptc,
We're interested in taking this change.  If you don't have time to look into possible msbuild flags, then let's get this change in for now.  It currently has a merge conflict with a meta file.  If you resolve that conflict, I can go ahead and complete this pull request for you.

Reply:
@AdamMitchell-ms, sorry for the delay. Was pulled into some other work the last two weeks. I can fix the merge conflict but there is still an issue with travis building the right DLLs for UWP because it is on an OSX environment.

I spent some time looking at Azure Pipeline builds and have a process that works well off of my own fork of this project. However, it requires that I grant access to this GitHub repo to the correct Azure pipeline account etc. Who can I work with to get that implemented for this project?

Reply:
It'll probably take roughly an hour on the phone with someone that has the right access to this repo and to the azure pipeline account we want to use

Reply:
@Khronoswebmaster , I think you have permissions to do what @ei2kpi-ptc is suggesting.  What is the normal process for making changes to branch policies?  I've approved the changes in this PR but haven't looked into the azure pipelines build.

Reply:
@ei2kpi-ptc, Is there any reason not to complete this PR in the meantime?  Is the travis build issue you mentioned a regression caused by this change?  Or already broken and still broken with your change?

If it's the latter (existing issue that's not fixed), then I'll go ahead and complete the PR.  If it's a regression, then I'll hold off for now.

Reply:
@AdamMitchell-ms  Yep, should be safe to accept the PR. 
The travis DLL issue is an existing issue which will be fixed once we get Azure Pipelines enabled in this repo (it is already enabled and working in my fork). This PR has travis.yml only building / releasing the .unitypackage version which always had the UWP DLLs missing.

@Khronoswebmaster I set up Azure Pipelines for the UPM package on my fork by using the existing wizard flow which is why I need to work with whoever has the right access to the Azure DevOps account for KhronosGroup. 
The yaml version of Azure Pipelines (which would be much more straightforward and similar to travis IMO) seems to have an existing issue with using secret GitHub credentials for the actual Release publishing step.

Reply:
Ok @ei2kpi-ptc , I've completed this PR now.